### DISCUSSION TITLE
能够指定运行的VW2路径吗

### DISCUSSION THREAD

**Discussion by tianya66** [tauri-apps/tauri#11879] [Q&A]
目前不需要安装程序，仅仅只需要一个可执行程序
就像 [wails](https://wails.io/docs/guides/windows/) 那样可以把固定版本的WV2一起打包进exe可执行程序里边 并且指定tauri使用该WV2

wails 配置方式
![image](https://github.com/user-attachments/assets/0857ec89-84da-452c-8343-38a9d8f16286)

**Comment by FabianLars** [comment]
fixed version dist is suppoert: https://v2.tauri.app/distribute/windows-installer/#fixed-version
setting the webview2 path uncoupled from that feature however only via the webview2 built-in `WEBVIEW2_BROWSER_EXECUTABLE_FOLDER` env var (can be set in rust above the tauri::Builder line if you want to ship that env var to your users).

**Reply by tianya66** [reply]
非常感谢
